butterknife-gradle-plugin/src/main/groovy/butterknife/plugin/ButterKnifePlugin.groovy

```diff
@@ -5,6 +5,7 @@ import com.android.build.gradle.LibraryPlugin
 import com.android.build.gradle.TestPlugin
 import com.android.build.gradle.api.BaseVariant
 import com.android.build.gradle.api.BaseVariantOutput
+import groovy.json.StringEscapeUtils
```

Comment:
These two can also work:
- [Option one](https://github.com/xirt4m/butterknife/commit/810b3bc04a0ad53863c210902c3728680908bcdb) with the [Matcher.quoteReplacement(String s)](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Matcher.html#quoteReplacement-java.lang.String-).
- [Option two](https://github.com/xirt4m/butterknife/commit/50fe0bb8405ac7bd0cc7a6772a25e4996779bac7) manual escape, if necessary.

Do you like one of them?